storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -897,11 +904,11 @@ struct my_option xb_client_options[] = {
      0},
 
     {"stream", OPT_XTRA_STREAM,
-     "Stream all backup files to the standard output "
-     "in the specified format. Currently supported formats are 'tar' and "
-     "'xbstream'.",
+     "Stream all backup files using xbstream format. Files are streamed to "
```

Comment:
right, value should be optional since we support only xbstream.
Am talking about REQUIRED_ARG to OPT_ARG

Reply:
It is already OPT_ARG